docs/src/pluto-simulation.md

```diff
@@ -0,0 +1,21 @@
+# MRI simulation in Pluto
+
+This example is extracted from the [Workshop MRI: Processing your Data](https://github.com/LIBREhub/MRI-processing-2023).
+
+Within that workshop, you'll find pertinent presentations on [open-source software](https://github.com/LIBREhub/MRI-processing-2023/blob/main/02-simulation/day1_OpenSoftware_Nov2023.pdf) and an [introduction to MRI simulation](https://github.com/LIBREhub/MRI-processing-2023/blob/main/02-simulation/day1_MRI_simulation_Nov2023.pdf).
+
+The following outlines tasks for simulating with **KomaMRI** in a **Pluto** notebook. We recommend attempting these tasks independently first and then reviewing the concealed solution.
+
+```@raw html
+<iframe type="text/html" src="../../../examples/pluto/simulation-example-empty.html" style="height:100vh;width:100%;"></iframe>
+```
+```@raw html
+<details><summary>View Solution</summary>
+<iframe type="text/html" src="../../../examples/pluto/simulation-example.html" style="height:100vh;width:100%;"></iframe>
+</details>
+```
+```@raw html
+<br>
+```
```

Comment:
```suggestion
# MRI simulation in Pluto

This example was used in the workshop [MRI: Processing your own data](https://github.com/LIBREhub/MRI-processing-2023). In the workshop's GitHub you'll find:
 -  A presentation on [open-source software](https://github.com/LIBREhub/MRI-processing-2023/blob/main/02-simulation/day1_OpenSoftware_Nov2023.pdf) 
 - and an [introduction to MRI simulation](https://github.com/LIBREhub/MRI-processing-2023/blob/main/02-simulation/day1_MRI_simulation_Nov2023.pdf).

The participants used **KomaMRI** within a **Pluto** notebook to simulate some basic 1D sequences:  FID, Gradient Echo (GE), and Spin Echo (SE). Try to solve the problems yourself! 😼

```@raw html
<iframe type="text/html" src="../../../examples/pluto/simulation-example-empty.html" style="height:100vh;width:100%;"></iframe>
```

```@raw html
<details><summary>View Solution</summary>
<iframe type="text/html" src="../../../examples/pluto/simulation-example.html" style="height:100vh;width:100%;"></iframe>
</details>
```
```

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/248/commits/ea2036a1a44166c4551226bd68e773cdfa0c4622